### PR TITLE
Comment unanonymous 2

### PR DESCRIPTION
**Pull request by dianawi**

### CODE REVIEW

File: system/datatypes/peer_reviewed_article.php [hunk 1]
```diff
@@ -1182,10 +1182,13 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
...
 			$email = __('art-email');
 			$emailFieldName = self::FIELD_NAME_DISCUSSION_COMMENTER_EMAIL;
+			$unanonymous =  __('art-comment-unanonymous');
 			$html .= <<<EOF
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<strong><label for="$nameFieldName"> $name </label></strong> <input type="text" id="$nameFieldName" name="$nameFieldName" />
+				<strong><label for="$nameFieldName"> $name $unanonymous </label>  </strong><input type="text" id="$nameFieldName" name="$nameFieldName" />
 			</div>
+EOF;
+			$html .= <<<EOF
```
Comment (matthijskooijman) [inline system/datatypes/peer_reviewed_article.php]: Why break the EOF-block here? IIRC this change is not needed?

Comment (matthijskooijman) [inline system/datatypes/peer_reviewed_article.php]: Why break the EOF-block here? AFAICS this change is not needed?

File: system/datatypes/peer_reviewed_article.php [hunk 2]
```diff
@@ -1182,10 +1182,13 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 			$nameFieldName = self::FIELD_NAME_DISCUSSION_COMMENTER_NAME;
 			$email = __('art-email');
 			$emailFieldName = self::FIELD_NAME_DISCUSSION_COMMENTER_EMAIL;
+			$unanonymous =  __('art-comment-unanonymous');
 			$html .= <<<EOF
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<strong><label for="$nameFieldName"> $name </label></strong> <input type="text" id="$nameFieldName" name="$nameFieldName" />
+				<strong><label for="$nameFieldName"> $name $unanonymous </label>  </strong><input type="text" id="$nameFieldName" name="$nameFieldName" />
```
Comment (matthijskooijman) [inline system/datatypes/peer_reviewed_article.php]: I wonder if this unanonymous remark should be in the `<label>` tag, or perhaps outside of it? I wonder if it is conceptually part of the label or not? Or should this perhaps be an info-icon?

File: system/languages/en.php [hunk 3]
```diff
@@ -257,6 +257,7 @@
 		"ml-successfully-unsubscribed" => "you have successfully unsubscribed",
 		"ml-successfully-created" => "successfully created",
 		"ml-successfully-updated" => "successfully updated",
+		"ml-comment-please-confirm" => "Thank you for your comment. Please confirm your identity by clicking the confirmation links that we have sent you by email.",
```
Comment (matthijskooijman) [inline system/languages/en.php]: This commit only seems to add this string, but never uses it?

**Comment by laurensmartina** [comment]
Fixed all, please review again.

**Comment by matthijskooijman** [comment]
Works, rebased and merging.

**Pull request merged**